### DISCUSSION TITLE
Installing Tabpy on Tableau Server

### DISCUSSION THREAD

**Discussion by dannikaaa** [tableau/TabPy#562] [Q&A]
Hi guys!

I was trying to install tabpy on tableau server and it crashed the whole server, I was wondering if you have more detailed instructions on which python version should be installed? And also more instructions on installing in on the server

Thanks!

**Comment by nmannheimer** [comment]
You should never install TabPy on the same machine as Tableau Server. This can cause multiple performance and security issues.

TabPy should be installed on a separate instance but besides that, Python 3.7-3.9 should work.
